operations/image_management/Import_External_Image_to_IMS.md

```diff
@@ -4,15 +4,28 @@ The Image Management Service \(IMS\) is typically used to build images from IMS
 However, it is sometimes the case that an image is built using a mechanism other than by IMS and needs to be added to IMS. In these cases,
 the following procedure can be used to add this external image to IMS and upload the image's artifact(s) to the Simple Storage Service (S3).
 
+* [Prerequisites](#prerequisites)
+* [Limitations](#limitations)
+* [Procedure](#procedure)
+    1. [Ensure supported format](#1-ensure-supported-format)
+    1. [Set helper variables](#2-set-helper-variables)
+    1. [Record artifact checksums](#3-record-artifact-checksums)
+    1. [Create image record in IMS](#4-create-image-record-in-ims)
+    1. [Upload artifacts to S3](#5-upload-artifacts-to-s3)
+    1. [Create, upload, and register image manifest](#6-create-upload-and-register-image-manifest)
+
 ## Prerequisites
 
-* System management services \(SMS\) are running in a Kubernetes cluster on non-compute nodes \(NCN\) and include the following deployments:
-  * `cray-ims`, the Image Management Service \(IMS\)
-* `kubectl` is installed locally and configured to point at the SMS Kubernetes cluster.
-* An image root archive or a pre-built image root SquashFS archive is available.
-* Optionally, additional image artifacts including a kernel, `initrd`, and kernel parameters file are available.
+* CSM is fully installed.
+* The Image Management Service \(IMS\) is healthy.
```

Comment:
Worth a pointer to the health checks?  Save the admin a search.

Reply:
Good call.